src/Apps/Collect2/Routes/Collection/Components/Header/index.tsx

```diff
@@ -180,11 +180,13 @@ export const CollectionHeader: FC<Props> = ({ artworks, collection }) => {
       {({ xs, sm, md, lg }) => {
         const size = xs ? "xs" : sm ? "sm" : md ? "md" : lg ? "lg" : "xl"
         const imageWidth = imageWidthSizes[size]
-        const imageHeight = xs ? imageHeightSizes.xs : imageHeightSizes.sm
+        const smallerScreen = size === "xs" || size === "sm"
```

Comment:
Is there a difference between this statement and the following?

```
const smallerScreen = xs || sm
```

If not, I'd prefer this ^^^, because it's less code to read.